transformations/butter_fingers_perturbation_for_IL/test.json

```diff
@@ -0,0 +1,185 @@
+{
+  "type": "butter_fingers_perturbation",
+  "test_cases": [
+    {
+      "class": "ButterFingersPerturbation",
+      "inputs": {
+        "language": "hi",
```

Comment:
The "language" parameter should go in args like this implementation: https://github.com/GEM-benchmark/NL-Augmenter/blob/main/transformations/mix_transliteration/test.json